### ISSUE TITLE
browserify module not loaded.

### ISSUE THREAD

**Issue by searls** [linemanjs/lineman-browserify#15]
> Also Note
> 
> I had to run npm install browserify --save-dev in addition to npm install lineman->browserify --save-dev. Otherwise I got an errors like this when I ran lineman run:
> 
> Loading "browserify.js" tasks...ERROR
> 
> > > Error: Cannot find module 'browserify'
> 
> Running "common" task
> Warning: Task "browserify" not found. Used --force, continuing.
> The README doesn't mention this extra step.

From @flyingzumwalt in #14 

This looks like an issue for @jasonkarns -- is it possible that this is still an issue?

**Comment by searls** [comment]
Hey @flyingzumwalt can you verify your lineman & lineman-browserify versions? I just tried this from scratch doing:

```
$ lineman new foo
$ cd foo
$ npm install --save-dev lineman-browserify
$ lineman run
```

And the browserify task worked fine. (It pulled down lineman-browserify 0.1.1)

**Comment by flyingzumwalt** [comment]
@searls I've got lineman-browserify 0.1.1, grunt-browserify 1.3.1, and  browserify 3.31.2 

browserify only installed when I explicitly installed it.  I don't know when grunt-browserify appeared.  I assume it was required by lineman-browserify.

**Comment by searls** [comment]
Yup, something is definitely wrong, but it's an erratic failure. I have not been able to reproduce, but @jasonkarns was (for only a moment, before it sorted itself)
